src/main/java/frc/trigon/robot/misc/objectdetectioncamera/ObjectDetectionCamera.java

```diff
@@ -21,6 +22,16 @@ public void periodic() {
         Logger.processInputs(hostname, objectDetectionCameraInputs);
     }
 
+    public void startTrackingObject() {
```

Comment:
Your logic is still flawed...

Reply:
Try to look at the code for the alignToNoteCommand, and actually think about the logic and how it should work.

Reply:
I looked at it, I don't understand what I'm missing.

Reply:
Create and send me a flow chart of how the logic works, and we'll think about it together